include/versioningCompiler/CompilerImpl/ClangLLVM/OptUtils.hpp

```diff
@@ -31,7 +31,19 @@
 
 // opt includes
 #include "NewPMDriver.h"
+#if LLVM_VERSION_MAJOR >= 18
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/TargetParser/Triple.h"
+#include "llvm/TargetParser/SubtargetFeature.h"
+#include "llvm/TargetParser/Host.h"
+#include "llvm/Passes/PassPlugin.h"
+#include "llvm/IR/PassManager.h"
+#else
 #include "llvm/ADT/Triple.h"
+#include "llvm/MC/SubtargetFeature.h"
+#include "llvm/Support/Host.h"
+#include "llvm/Passes/PassBuilder.h"
+#endif
```

Comment:
in the other file you used `< 18` as comparison, can you be consistent, i.e., use `< 19`?